src/v/redpanda/admin/api-doc/migration.def.json

```diff
@@ -42,6 +42,31 @@
         }
     }
 },
+"outbound_topic": {
+    "type": "object",
+    "required": [
+        "topic"
+    ],
+    "properties": {
+        "topic": {
+            "type": "string",
+            "description": "Topic name"
+        },
+        "ns": {
+            "type": "string",
+            "description": "Topic namespace. If not present it is assumed that topic is in the \"kafka\" namespace"
+        },
+        "remote_topic": {
+            "description": "Original topic name (at the time data was first written to cloud storage).",
+            "type": "namespaced_topic",
+            "$ref": "namespaced_topic"
+        },
+        "remote_location": {
+            "description": "Location hint of the form <original cluster uuid>/<original revision id> used to disambiguate different topic instances.",
+            "type": "string"
+        }
```

Comment:
Using `topic/uuid/rev` format for inbound migrations and for discovery API was a requirement from Matt: a single compound identifier can at least be copy&pasted easily. I think it'd be better to return it here in the same format: `remote_topic_name/cluster_uuid/initial_rev`. Remote topic namespace is not of particular use, as it's always `kafka`. WDYT?

Reply:
Indeed looks easier for the API client if they can just stick a string into `source_topic_location` of the inbound topic.